test/wanda/catalog_test.exs

```diff
@@ -404,6 +374,56 @@ defmodule Wanda.CatalogTest do
              |> Enum.any?(&(&1.name == "non_existing_value"))
     end
 
+    test "should properly compute customizability information" do
+      selectable_checks =
+        Catalog.get_catalog_for_group(Faker.UUID.v4(), %{
+          "id" => "mixed_values_customizability"
+        })
+
+      find_check = fn id ->
+        Enum.find(selectable_checks, fn %SelectableCheck{id: check_id} ->
+          id == check_id
+        end)
+      end
+
+      assert %SelectableCheck{customizable: false, values: forcedly_non_customizable_values} =
+               find_check.("non_customizable_check")
+
+      assert Enum.all?(forcedly_non_customizable_values, fn %{customizable: customizable} ->
+               not customizable
+             end)
+
+      assert %SelectableCheck{customizable: false, values: []} =
+               find_check.("check_without_values")
+
+      %SelectableCheck{customizable: false, values: explicit_non_customizable_check_values} =
```

Comment:
nitpick: maybe put an assert here to have the same in all entries

Reply:
thanks done here bcdd0627019dcb96cce90fd74244facf175551b1